ratis-server/src/main/java/org/apache/ratis/server/storage/StorageImplUtils.java

```diff
@@ -130,13 +131,14 @@ RaftStorageImpl run() throws IOException {
       }
     }
 
+    @SuppressWarnings("java:S1181")
```

Comment:
Could you add a comment at the end for this and the other SuppressWarnings?
```java
@SuppressWarnings("java:S1181") // catch Throwable
```
```java
@SuppressWarnings("java:S2095") // return Closable
```

Reply:
OK